src/main/java/calculator/calculate/CalculatorService.java

```diff
@@ -0,0 +1,21 @@
+package calculator.calculate;
+
+import calculator.input.Numbers;
+import calculator.input.Symbols;
+public class CalculatorService {
+
+    public CalculatorService() {
+    }
+
+    public int calculate(Numbers numbers, Symbols symbols) {
+        int value = numbers.getNumbers().get(0).getNumber();
```

Comment:
일급 컬렉션을 사용할꺼라면 좀 더 적극적으로 활용해봐도 좋을 것 같아요.
```suggestion
        int value = numbers.getFirst();
```